boards/arm/thingy53_nrf5340/Kconfig.defconfig

```diff
@@ -103,6 +103,58 @@ config REGULATOR
 
 endif # !TRUSTED_EXECUTION_SECURE
 
+if BOARD_SERIAL_BACKEND_CDC_ACM
+
+config USB_DEVICE_PRODUCT
+	default "Thingy:53 Application"
+
+config USB_DEVICE_VID
+	default 0x1915
+
+config USB_DEVICE_PID
+	default 0x530C
+
+config USB_DEVICE_STACK
+	default y
+
+config USB_CDC_ACM
+	default y
+
+config UART_CONSOLE
+	default CONSOLE
+
+config USB_DEVICE_INITIALIZE_AT_BOOT
+	default y if !MCUBOOT
+
+config SHELL_BACKEND_SERIAL_CHECK_DTR
+	default SHELL
+
+config USB_DEVICE_REMOTE_WAKEUP
+	default n
+
+if LOG
+
+# Logger cannot use itself to log
+config USB_CDC_ACM_LOG_LEVEL
+	default 0
+
+# Set USB log level to error only
+config USB_DEVICE_LOG_LEVEL
+	default 1
```

Comment:
Quickly looking at the implementation, I noticed that it most probably should not lead to any unexpected behavior (but it could lead to undefined behavior if logger implementation changes)

Reply:
Added change in upstream PR https://github.com/zephyrproject-rtos/zephyr/pull/54488 and cherrypicked as fromlist